app/views/releases/index.html.erb

```diff
@@ -27,7 +26,9 @@
 
       <%= paginate @releases %>
     <% else %>
-      <p>There haven't been any releases yet.</p>
+      <div class="alert alert-warning">
+        <p>There haven't been any releases yet: <%= link_to "create the first release", new_project_release_path(@project) %>.</p>
```

Comment:
maybe mention `add a release branch on <link to stage settings> to automatically create releases`